### ISSUE TITLE
Build fail after updating "rocksdb" and "netty-tcnative-boringssl-static" package

### ISSUE THREAD

**Issue by odidev** [apache/bookkeeper#2378]
**BUG REPORT**

***Describe the bug***
After updating the latest version of netty-tcnative-boringssl-static (2.0.31.Final) and rocksdb(6.10.2) package in pom.xml, build of bookkeeper-server package is failing due to compilation warning. 
And after removing "Werror" option from compilerArguments, build passed successfully. 

***To Reproduce***

1. Update netty-tcnative-boringssl-static  to "2.0.31.Final" in pom.xml
2. Update rocksdb to "6.10.2" in pom.xml
3. Build the bookkeeper-server package by executing the "mvn -pl bookkeeper-server install -DskipTests" command
4. See error:

```
[WARNING] COMPILATION WARNING :
[INFO] -------------------------------------------------------------
[WARNING] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java:[128,24] setMaxBackgroundCompactions(int) in org.rocksdb.Options has been deprecated
[WARNING] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java:[129,24] setMaxBackgroundFlushes(int) in org.rocksdb.Options has been deprecated
[WARNING] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java:[138,29] setBlockCacheSize(long) in org.rocksdb.BlockBasedTableConfig has been deprecated
[WARNING] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java:[142,33] setFilter(org.rocksdb.Filter) in org.rocksdb.BlockBasedTableConfig has been deprecated
[INFO] 4 warnings
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java: warnings found and -Werror specified
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  12.116 s
[INFO] Finished at: 2020-07-15T08:36:35Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.1:compile (default-compile) on project bookkeeper-server: Compilation failure
[ERROR] /netty/bookkeeper/bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/storage/ldb/KeyValueStorageRocksDB.java: warnings found and -Werror specified
```

***Expected behavior***
Build should pass.

***Additional context***
Latest version of netty-tcnative-boringssl-static (2.0.31.Final) and rocksdb(6.10.2) packages are having ARM64 support.
This issue is applicable for aarch64 and amd64 platforms both.

**Comment by eolivelli** [comment]
Please also update netty.
The error means that they deprecated one method and you should use the new method, check the Docs of the new version of that library

**Comment by odidev** [comment]
This is fixed in #2379
